network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreIT.java

```diff
@@ -57,7 +61,7 @@ public void test() {
             // import new network in the store
             assertTrue(service.getNetworkIds().isEmpty());
             Network network = service.importNetwork(new ResourceDataSource("test", new ResourceSet("/", "test.xiidm")));
-            service.flush(network);
+//            service.flush(network);
```

Comment:
You can remove the line

Reply:
I commented the line for test purpose and forgot to uncomment it. I'll better keep the line uncommented.